docs/DeveloperGuide.md

```diff
@@ -665,6 +665,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ---
 
+**Use case: Access a student's record through a filter**
+
+**MSS**
+
+1. User request to access a student's record using their name, class and category to change.
+2. EduTrack shows the category and requests for new input in the category.
+3. User enters the new input for the category.
+4. EduTrack updates and displays the new student record.
+
+Use case ends.
+
+**Extensions**
+* 1a. Student name, class name or category is not specified.
+  * 1a1. EduTrack informs that student name, class name or category is missing.
+
+    Use case ends.
+* 1b. EduTrack detects that student name, class name or category does not exist.
+  * 1b1. EduTrack informs the user that the student name, class name or category does not exist.
+  * 1b2. EduTrack terminates the request.
+
+    Use case ends.
+
```

Comment:
I like the extensions given, maybe you could write them in similar style. ie 1a. starts with "EduTrack detecting missing inputs" just like 1b. It also follows the format of starting with either the Actor or the System :)

Reply:
I have changed the wording of the use case to better reflect the requirements of a use case. Let me know if there is anything else that looks out of place 😄